docs/UserGuide.md

```diff
@@ -259,182 +277,191 @@ Format: `find PREFIX KEYWORD [KEYWORD]…​`
 
 ## Tag Features
 
-### Adding a tag : `create-tag`
+### Creating a tag : `create-tag` or `ctag`
 
 Creates a `Tag` within WedLinker to be used on contacts.
 
-Format: `create-tag t/TAGNAME`
+Format: `create-tag t/TAG_NAME` or`ctag t/TAG_NAME`
 
-* The `TAGNAME` is alphanumeric and can contain whitespaces.
+* The `TAG_NAME` is alphanumeric and can contain whitespaces.
+* `TAG_NAME` is case-insensitive.
```

Comment:
A bit of duplication with the line below where it says 'Tags are case-insensitive, so you cannot have both...' - can maybe remove that line and consider adding elaboration here, not sure if the elaboration is necessary

Reply:
👍